yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -141,3 +148,16 @@ func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*s
 	}
 	return nil
 }
+
+func (conv *DebeziumValueConverter) GetTableNameToSchema() map[string]map[string]map[string]string {
+	//need to create explicit map with required details only as can't use TableSchema directly in import area because of cyclic dependency
```

Comment:
Let's discuss this once on call? I want to understand the cyclic dependency once.

Reply:
yes sure

Reply:
pls add a TODO here to fix